### PR TITLE
Update aXe core and add Core 3.1 to nuget package

### PR DESCRIPTION
**Pull request by TroyWalshProf**
#22 - Update Dotnet Core and aXe core

### CODE REVIEW

File: Selenium.Axe/Selenium.Axe/Selenium.Axe.csproj [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net45;net471;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net45;net471;netstandard2.0;netcoreapp3.1</TargetFrameworks>
```
Comment (dbjorge) [inline Selenium.Axe/Selenium.Axe/Selenium.Axe.csproj]: Adding this explicitly to the test project is good for the extra test coverage, but I'm not sure it's necessary in the main project; I'd have expected it to already be 3.1 compatible based on the netstandard2.0 target. Is there something new in 3.1 motivating the specific target for netcore?

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 Automated web accessibility testing with .NET, C#, and Selenium. Wraps the [axe-core](https://github.com/dequelabs/axe-core) accessibility scanning engine and the [Selenium.WebDriver](https://www.seleniumhq.org/) browser automation framework.
 
-Compatible with .NET Standard 2.0+, .NET Framework 4.5+, and .NET Core 2.0+.
+Compatible with .NET Standard 2.0+, .NET Framework 4.5+, and .NET Core 3.1+.
```
Comment (dbjorge) [inline README.md]: Is this change intentional? I think the original is still more about, since it doesn't look like you're removing .NET Core [2.0, 3.1) support

**Comment by dbjorge** [comment]
(+ the failing CI build)

File: README.md (hunk 2 above)
Comment (TroyWalshProf) [reply README.md]: You are correct

File: Selenium.Axe/Selenium.Axe/Selenium.Axe.csproj (hunk 1 above)
Comment (TroyWalshProf) [reply Selenium.Axe/Selenium.Axe/Selenium.Axe.csproj]: Again correct :)

**Comment by TroyWalshProf** [comment]
Fixed build issue, for some reason the 2.2 and 3.1 tests don't like to play together.
